python/ray/resource_spec.py

```diff
@@ -166,9 +166,14 @@ def resolve(self, is_head, node_ip_address=None):
             if gpu_ids is not None:
                 num_gpus = min(num_gpus, len(gpu_ids))
 
-        info_string = _get_gpu_info_string()
-        gpu_types = _constraints_from_gpu_info(info_string)
-        resources.update(gpu_types)
+        gpu_types = {}
+        try:
+            info_string = _get_gpu_info_string()
+            gpu_types = _constraints_from_gpu_info(info_string)
+        except BaseException:
```

Comment:
```suggestion
        except Exception:
```

Don't catch base exception.